clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/providers/TitusClusterProvider.groovy

```diff
@@ -279,9 +322,13 @@ class TitusClusterProvider implements ClusterProvider<TitusCluster>, ServerGroup
     healths.each { healthEntry ->
       def instanceId = healthKeysToInstance.get(healthEntry.id)
       healthEntry.attributes.remove('lastUpdatedTimestamp')
-      instances[instanceId].health << healthEntry.attributes
+      def health = healthEntry.attributes.collectEntries { key, value ->
```

Comment:
I'm not sure if changing this value to an explicit string here (instead of a boolean) will affect the UI. I am able to run tasks locally with Orca + Clouddriver (create/clone/distroy).